sdk/aqueduct/client.py

```diff
@@ -479,6 +479,7 @@ def publish_flow(
         k_latest_runs: Optional[int] = None,
         config: Optional[FlowConfig] = None,
         source_flow: Optional[Union[Flow, str, uuid.UUID]] = None,
+        run_now: bool = True,
```

Comment:
nit: Can you add a comment below documenting how this should be used by users since this is the doc string that they will access.